src/interface/SPARQLInterface.ts

```diff
@@ -42,7 +43,7 @@ export async function fetchConcepts(
         "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>",
```

Comment:
Consider refactoring to some text file with variables, or using some SPARQL builder https://www.npmjs.com/package/@atomgraph/sparql-builder